src/server/inline.preboot.code.ts

```diff
@@ -19,12 +19,11 @@ export const defaultOptions = <PrebootRecordOptions>{
     },
     {selector: 'select,option', events: ['change']},
 
-    // when user hits return button in an input box
+    // when user submit form (press enter, click on button/input[type="submit"])
     {
-      selector: 'input',
```

Comment:
It looks like you removed the default for when someone hits the return key in an input box. I agree with you that when someone has an input box within a form tag, what you have will work. However, not everyone actually uses a form tag these days and they still need the case where you hit the return key from within an input box that is not part of an HTML form.

Would it work for you to keep the old listener and add a new one for form submit?